### PR TITLE
reading each pixel from bmp

### PR DESCRIPTION
**Pull request by TschantzN**
has we discussed here is the class extracting the width, height bitsperpixel and the pixels

### CODE REVIEW

**Review by GianniCecchetto — COMMENTED** [review]
Why is there a method to write in the BmpReader class ?

File: ImageToChar/src/main/java/ch/heigvd/dai/bmpios/BmpReader.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    int red = pixelData[index + 2] & 0xFF;
+                    System.out.printf("Pixel at (%d, %d): R=%d, G=%d, B=%d%n", x, y, red, green, blue);
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public int writeText(String fileName,String content){
```
Comment (GianniCecchetto) [inline ImageToChar/src/main/java/ch/heigvd/dai/bmpios/BmpReader.java:59]: Should be in another class BmpWriter

Comment (GianniCecchetto) [inline ImageToChar/src/main/java/ch/heigvd/dai/bmpios/BmpReader.java:59]: This method should be in another class like BmpWriter

**Review by GianniCecchetto — CHANGES_REQUESTED** [review]

File: ImageToChar/src/main/java/ch/heigvd/dai/bmpios/BmpReader.java (hunk 1 above)
Comment (TschantzN) [reply ImageToChar/src/main/java/ch/heigvd/dai/bmpios/BmpReader.java:59]: absolutely, it will be done tomorrow morning

**Pull request merged**